src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -116,6 +116,13 @@ object ExtractionPipeline {
       linkedField = "dataset"
     )
 
+    // don't need to use targets apart from storing them, so we don't assign an output here
+    extractLinkedEntities(
+      entityToExtract = EncodeEntity.Target,
+      matchingField = "target",
+      linkingEntities = experiments
```

Comment:
Just wondering - when "linkedField" is not provided, it will default to using the @id, correct?
A "type=Target" query has the following in the id field:
**"@id": "/targets/ztf-8-celegans/"**

Reply:
Yeah, that sounds right. And the target "@id" field should match up with the experiment "target" field.